WorkloadManagementSystem/Agent/SiteDirector.py

```diff
@@ -973,16 +990,16 @@ def _writePilotScript( self, workingDirectory, pilotOptions, proxy = None,
   os.environ["X509_CERT_DIR"]=os.path.join(pilotWorkingDirectory, 'etc/grid-security/certificates')
   # TODO: structure the output
   print '==========================================================='
-  print 'Environment of execution host'
+  logger.debug('Environment of execution host\n')
   for key in os.environ.keys():
     print key + '=' + os.environ[key]
```

Comment:
Why do you keep a `print` here? if the level is not debug the previous comment will not be printed but the environment will be dumped... use `logger.debug()` here as  well?

Reply:
I kept a print here because it's easier visualized, like this:

```
===========================================================
2017-11-16 10:54:21 UTC DEBUG    Environment of execution host

LESS=-R
UPSTART_EVENTS=xsession started
XDG_SESSION_TYPE=x11
LC_PAPER=de_CH.UTF-8
...
```
The logging level is set to debug few lines before.